### PR TITLE
Added Resons necesary on ban command.

### PR DESCRIPTION
**Pull request by NycuRO**

### CODE REVIEW

File: src/pocketmine/command/defaults/BanCidByNameCommand.php [hunk 1]
```diff
@@ -68,6 +68,11 @@ public function execute(CommandSender $sender, $currentAlias, array $args){
 
 			return false;
 		}
+		
+		if(count($args) === 1){
+			$sender->sendMessage("Please insert reason why them need ban-cid by name.");
```
Comment (erkam2002) [inline src/pocketmine/command/defaults/BanCidByNameCommand.php]: Please insert reason why

Comment (NycuRO) [inline src/pocketmine/command/defaults/BanCidByNameCommand.php]: ?

Comment (NycuRO) [inline src/pocketmine/command/defaults/BanCidByNameCommand.php]: This block ban command used without reason.

Command need use like that: /ban name reason.

This block command: /ban name .

**Comment by MasterOfTheRealm** [comment]
@Leo3418 is right, It can be annoying to provide a reason for banning someone but at the same time, useful to keep track of staff bans.

**Comment by NycuRO** [comment]
@Leo3418 Kick and ban :P

**Comment by Leo3418** [comment]
@NycuRO no matter what I'll not accept this until users can choose to add this enforcement. We are developing software, not teaching users how to be an OP. The bottom line would be something like this (when you run `sudo` for the first time in CentOS terminal):
new image - nmsbo

**Comment by DerddyDert** [comment]
@NycuRO can we get options for this put in the Genisys.yml ?

File: src/pocketmine/command/defaults/BanCidByNameCommand.php (hunk 1 above)
Comment (zKoz210) [inline src/pocketmine/command/defaults/BanCidByNameCommand.php]: It's stupid if you need it, make your own plugin

**Comment by NycuRO** [comment]
@DerddyDert This is my part. If you don't want, close PR or you can make that :P

**Comment by DerddyDert** [comment]
Thanks @NycuRO

**Pull request closed without merging**